src/handlers/vcs-handlers/GitHandler.js

```diff
@@ -144,9 +144,11 @@ class GitHandler {
             );
         }
 
+        const kapetaReleaseBranch = process.env.KAPETA_RELEASE_BRANCH;
+
         return {
             branch,
-            main: defaultBranch === branch,
+            main: defaultBranch === branch || kapetaReleaseBranch === branch,
```

Comment:
It's not strictly "main" is it? 🤔  I get we want to make a release, but this is a little confusing.